Makefile

```diff
@@ -122,7 +122,7 @@ ETCD=$(TOOLS_BIN_DIR)/etcd
 # Define Docker related variables. Releases should modify and double check these vars.
 REGISTRY ?= gcr.io/$(shell gcloud config get-value project)
 STAGING_REGISTRY := gcr.io/k8s-staging-cluster-api-azure
```

Comment:
seems strange that we'd keep using gcr.io for staging, did you find any info about it in the related issues?

Reply:
When creating the v1.3.2 CAPZ release, I couldn't find a staging URL under registry.k8s.io that worked, only the existing gcr.io one in the Makefile. I haven't found any specific guidance for staging except what KEP 3000 says under "Non-Goals":
> Anything related to creation of artifacts, bom, staging buckets.

Reply:
ok let's keep it for now and we can change it later if there is a new staging registry